COLLABORATOR_GUIDE.md

```diff
@@ -230,6 +247,11 @@ Time to push it:
 ```text
 $ git push origin master
 ```
+* Optional: Force push the amended commit to the branch you used to open the pull request. If your branch is called `bugfix`, then the command would be `git push --force-with-lease origin master:bugfix`. When the pull request is closed, this will cause the pull request to show the purple merged status rather than the red closed status that is usually used for pull requests that weren't merged. Only do this when landing your own contributions.
```

Comment:
You text made it sound like if you close a PR using the "close pull request" button that it would show as merged, but you mean "closing a PR in the way node does it"